review-tools/gitaddrev

```diff
@@ -103,18 +123,25 @@ foreach (@ARGV) {
     } elsif (/^--rmreviewers$/) {
         $rmrev = 1;
     } elsif (/^--myemail=(.+\@.+)$/) {
-	try_add_reviewer($1);
+        push @collect_possible_reviewers, $1;
     } elsif (/^--verbose$/) {
 	$verbose = 1;
     } elsif (/^--web$/) {
         $WHAT = 'web';
+        $min_authors = 1;
+        # openssl/web is governed by OMC
+        $min_otc = 0;
+        $min_omc = 1;
```

Comment:
For web both reviewers must be OMC (one of which may be the author) - at least I think that's the rules?? So shouldn't this be `$min_omc = 2`?

Reply:
Ah, correct, the OMC is expected to be the authorized reviewers.
(if someone else wants to review from a purely technical standpoint, that's ok too, of course)

I'll fix it as you suggest

Reply:
Fixed